apify-docs-theme/src/theme/custom.css

```diff
@@ -164,6 +165,8 @@ html[data-theme='dark'] {
     --ifm-h5-font-size: 1.4rem;
     --ifm-h6-font-size: 1.2rem;
 
+    --ifm-heading-line-height: 
```

Comment:
FYI this one is missing a value

Reply:
My bad, this is still in progress so you can just ignore the changes until we request the review again.

CC @webrdaniel